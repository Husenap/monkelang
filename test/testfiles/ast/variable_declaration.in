monke has number x is 42
monke has number y is 1337
