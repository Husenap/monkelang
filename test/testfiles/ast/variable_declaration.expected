IdDeclStmt: id:x type:number
  NumberExpr: 42
IdDeclStmt: id:y type:number
  NumberExpr: 1337
